Sources/FINNFilterImplementation/BackendModel/FilterData.swift

```diff
@@ -27,6 +27,20 @@ struct FilterData: Decodable {
         queries = try container.decodeIfPresent([FilterDataQuery].self, forKey: CodingKeys.queries) ?? []
     }
 
+    public func filterNode() -> CCFilterNode {
+        if let isRange = isRange, isRange {
+            let rangeNode = CCRangeFilterNode(title: title, name: parameterName)
```

Comment:
Why not just return here?

Reply:
I had a line in between at some point, I'll fix this